backend/src/main/kotlin/blue/mild/covid/vaxx/dto/AnswerDto.kt

```diff
@@ -0,0 +1,8 @@
+package blue.mild.covid.vaxx.dto
+
+import java.util.UUID
+
+data class AnswerDto(
+    val questionId: UUID,
```

Comment:
Jenom premyslim, jak budes vracet ty otazky, ktere se zobrazi a jak na to namapujes ty odpovedi. Tj. jestli by nestalo za to v tomhle DTO posilat rovnou i otazku.

Reply:
to je asi otazka na @misslecter jak by se ji libilo, aby to vypadalo, podle ceho si to bude mapovat a tak, pro me to neni problem

Reply:
texty bych nechala na FE i vzhledem k prekladum do budoucna. Takhle ten objekt je za me fajn

Reply:
nope, ackoliv ted se tvarime, ze ty otazky jsou fixni, tak do budoucna byt nemusi, budou chodit z backendu, lezi tam DTO `QuestionDto` co prijde z endpointu `/api/question` v nem je text + pripadny preklad

tzn bude trebat je mapovat

Reply:
joo uz chapu, tak kdyz budou propojeny pred `questionId`, tak je to fajn. Vypisu to, co mi prijde z `/api/question`, ulozim k nim `value`, a pak poslu `questionId` a `value` na BE.